src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -30,8 +31,8 @@ const Content = styled.div`
 `;
 
 const PageWrapper = ({ children, pageData, status }) => {
-  const { fonts: fontFunctions, service } = useContext(ServiceContext);
-  const fonts = fontFunctions.map(getFonts => getFonts());
+  const { service } = useContext(ServiceContext);
+  const { variant } = useContext(RequestContext);
```

Comment:
I think `variant` is available through `ServiceContext` as well. Not sure why its exposed through both contexts though 🤔

Reply:
Seen some scenarios where `variant` is undefined or different depending on which context you get it from

This only happens on tests or storybook though where people are importing `<ServiceContextProvider>` and `<RequestContextProvider>` and then passing their own arguments to them - eg, they provide a `variant` to one component, but not the other